pkgs/tools/filesystems/xfsdump/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchgit
+, attr
+, gettext
+, autoconf
+, automake
+, ncurses
+, libtool
+, libuuid
+, libxfs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xfsdump";
+  version = "3.1.10";
+
+  src = fetchgit {
+    url = "git://git.kernel.org/pub/scm/fs/xfs/xfsdump-dev.git";
+    rev = "3607469d0a8ca3513bfb6093a0da9932dddc8979";
```

Comment:
You could use the tag ([v3.1.10](https://git.kernel.org/pub/scm/fs/xfs/xfsdump-dev.git/tag/?h=v3.1.10)) here